File: language-basics/operators/arithmetic.go
package main;
import "fmt";

func main() {
	var a int = 21;
	var b int = 10;
	var c int;

	c = a + b;
	fmt.Printf("Sum of 21 and 10 is: %d\n", c);
	c = a - b;
	fmt.Printf("Differnce of 21 and 10 is : %d\n", c);
	c = a * b;
	fmt.Printf("Multiplication of 21 and 10 is : %d\n", c);
	c = a / b;
	fmt.Printf("Division of 21 and 10 is : %d\n", c);
	c = a % b;
	fmt.Printf("Remainder when 21 divided by 10 is : %d\n", c);
	a++;
	fmt.Printf("Increment a by 1 using increment operator:%d\n", a);
	a--;
	fmt.Printf("Decrement a by 1 using decrement operator:%d\n", a);
	
}

File: language-basics/operators/arithmetic.c
#include<stdio.h>

int main() {
    int a = 21;
    int b = 10;
    int c;

    c = a+b;
    printf("Sum of 21 and 10 is %d\n", c);
    c = a-b;
    printf("Difference of 21 and 10 is %d\n", c);
    c = a*b;
    printf("Multiplication of 21 and 10 is %d\n", c);
    c = a/b;
    printf("Division of 21 and 10 is %d\n", c);
    c = a%b;
    printf("Remainder of 21 divided by 10 is %d\n", c);
    c = a++;
    printf("increment value of a by 1 using increment operator %d\n", a);
    c = a--;
    printf("decrement value of a by 1 using decrement operator %d\n", a);
}